elements/rh-blockquote/rh-blockquote.ts

```diff
@@ -0,0 +1,74 @@
+import { LitElement, html, css, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { pfelement } from '@patternfly/pfe-core/decorators.js';
+
+import styles from './rh-blockquote.css';
+
+/**
+ * A blockquote for displaying quote, author, and author title.
+ *
+ *
+ * @summary  A blockquote for displaying quote, author, and author title.
+ *
+ *
+ * @slot quote   - Provide a quote for the blockquote
```

Comment:
Giving this some thought, does this need to be named?

Consider:
```html
<rh-blockquote>
  <p>In open source, we feel strongly that to really do something well, you have to get a lot of people involved.</p>
  <span slot="author">Linus Torvalds</span>
  <span slot="title">Software Engineer</span>
</rh-blockquote>
```

Reply:
That makes sense, I've updated it!